bucket_4D/xlockmore/manifests/plist.single

```diff
@@ -0,0 +1,8 @@
+@(root,,4111) bin/play.sh
+@(root,,4111) bin/xlock
```

Comment:
why use "@(root,,4111)" instead of "@mode(4111)"?  Is that just copied from freebsd? 
I'm guessing it's because ports allows users to build packages which is not done in ravenports (so owner is already root I think)

Reply:
That's what `ravenadm dev sort` made when I copied FreeBSD's plist.

Reply:
Ah, I see.  I think you just delete "root" in these two lines, and leave it as "@(,,4111) ".  dev sort command does eliminate single @mode commands, I remember now.